cmd/goal/application.go

```diff
@@ -46,8 +46,9 @@ var (
 	approvalProgFile string
 	clearProgFile    string
 
-	method     string
-	methodArgs []string
+	method         string
+	methodArgs     []string
+	methodCreation bool
```

Comment:
This should really be "appCreation" or "contractCreation", right?

Reply:
Well I meant it to mean creation for the method subcommand

Reply:
`methodAppCreation`? (it's always hard to find a good name)

Reply:
My objection is that when you read any code that uses it, it reads wrong.  It'll say, essentially
```
if methodCreation {
 // create the app
}
```
As with the other comment, whatever you want to go with is fine.  Let me know when to merge.

Reply:
Good points, I'll rename to `methodCreatesApp`